admin/satellites/geo-dns.md

```diff
@@ -0,0 +1,120 @@
+---
+title: GeoDNS
+description: |
+    Learn how to configure your primary and satellite deployments to share a
+    single hostname.
+state: alpha
+---
+
+Out of the box, satellites deployments have a different access URL to the
+primary deployment. This may cause confusion for engineering teams located
+globally. This may cause confusion amongst users wondering which access URL they
+should access Coder from.
+
+Coder supports a GeoDNS configuration where all satellites and the primary
+deployment share the same hostname, so all users access Coder using a consistent
```

Comment:
needs a passage about users being routed to the nearest installation or satellite, ensuring low latency